app/serializers/api/order_serializer.rb

```diff
@@ -16,7 +16,7 @@ def outstanding_balance
     end
 
     def payments
-      object.payments.joins(:payment_method).completed
+      object.payments.joins(:payment_method).where('state IN (?)', %w(completed pending))
```

Comment:
just a matter of styling, I believe `.where(state: %w(completed pending))` would also work and doesn't require you to get down to SQL. It's a matter of preference, so feel free to dismiss.

Reply:
Now that I checked again, we need payments in `pending` state so we can show them as requiring auth in /account, right?

Reply:
Yep, that's correct.